packages/neuron-ui/src/components/NetworkEditorDialog/hooks.ts

```diff
@@ -9,6 +9,7 @@ export const useOnSubmit = ({
   id = '',
   name = '',
   remote = '',
+  type,
```

Comment:
the field `type` is too common in CKB, it would be better to change it to `connectedClientType`